### DISCUSSION TITLE
Intergration not working with saver

### DISCUSSION THREAD

**Discussion by schijven** [xirixiz/homeassistant-afvalwijzer#370] [Show and tell]
Dear,

I am a fan of your afvalwijzer. Now changed my wastprovicer from "mijnafvalkalender" to "saver". I changed this in the setting, but I get the following error:

Deze fout is ontstaan door een aangepaste integratie.

Logger: custom_components.afvalwijzer.const.const
Bron: custom_components/afvalwijzer/sensor.py:52
integratie: Afvalwijzer (documentatie, problemen)
Eerst voorgekomen: 08:06:58 (1 gebeurtenissen)
Laatst gelogd: 08:06:58

No discovery information provided; sensors cannot be created.

What can I do to fix this?

Is it posible to run the code from pycharm for example?

**Comment by PatrickSt1991** [comment]
@schijven do you still have this error?

What version do you run and have you tried removing all the entities and start fresh via ***Item Toevoegen*** ?

**Comment by schijven** [comment]
I don't have any issue's annymore.
